### PR TITLE
created controller class, started working on configuring swagger

### PR DESCRIPTION
**Pull request by volodymyrruso**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -53,6 +53,31 @@
...
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger-ui</artifactId>
+            <version>3.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-bean-validators</artifactId>
+            <version>2.9.2</version>
+        </dependency>
```
Comment (liliia-ponomarenko) [inline pom.xml]: You can remove it and use springdoc-openapi-ui-starter

File: src/main/java/org/mate/bookstorespringboot/controller/BookController.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    public List<BookDto> getBooks() {
+        return bookService.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public BookDto getBookById(@PathVariable Long id) {
+        return bookService.findById(id);
+    }
+
+    @PostMapping
```
Comment (liliia-ponomarenko) [inline src/main/java/org/mate/bookstorespringboot/controller/BookController.java:37]: Let’s  add response status Created

File: src/main/java/org/mate/bookstorespringboot/controller/dto/BookRequestDto.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package org.mate.bookstorespringboot.controller.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookRequestDto {
```
Comment (liliia-ponomarenko) [inline src/main/java/org/mate/bookstorespringboot/controller/dto/BookRequestDto.java]: better use records for Dto

File: src/main/java/org/mate/bookstorespringboot/service/interfaces/BookService.java [hunk 4]
```diff
@@ -1,22 +1,32 @@
...
+
+    /**
+     * Find book by id.
+     *
+     * @param  id  id of the book
+     * @return     the book with the given id
+     */
+    BookDto findById(Long id);
 }
+
```
Comment (liliia-ponomarenko) [inline src/main/java/org/mate/bookstorespringboot/service/interfaces/BookService.java]:
```suggestion
```

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! A few minor comments ;)

File: src/main/java/org/mate/bookstorespringboot/controller/BookController.java (hunk 2 above)
Comment (volodymyrruso) [reply src/main/java/org/mate/bookstorespringboot/controller/BookController.java:37]: I will be using ResponseEntity in controller layers

**Pull request merged**